DependencyInjection/Configuration.php

```diff
@@ -83,6 +83,8 @@ public function getConfigTreeBuilder()
                             ->defaultValue('auto')
                         ->end()
                         ->scalarNode('formatter')->defaultNull()->end()
+                        ->booleanNode('inject_debugger_for_discovered_http_client')->defaultTrue()->end()
```

Comment:
Correct me if I am wrong, but this will register the default client in the discovery, will it not? If so, all the problems I listed in the previous PR can still occur. I would rather have this as an optional feature stating you can turn this on for discovered clients if you want.